op-dispute-mon/mon/extract/caller_test.go

```diff
@@ -89,7 +93,7 @@ func TestMetadataCreator_CreateContract(t *testing.T) {
 
 func setupMetadataLoaderTest(t *testing.T, gameType uint32) (*batching.MultiCaller, *mockCacheMetrics) {
 	fdgAbi := snapshots.LoadFaultDisputeGameABI()
-	if gameType == uint32(faultTypes.SuperPermissionedGameType) || gameType == uint32(faultTypes.SuperCannonGameType) {
+	if gameType == uint32(faultTypes.SuperPermissionedGameType) || gameType == uint32(faultTypes.SuperCannonGameType) || gameType == uint32(faultTypes.SuperAsteriscKonaGameType) {
 		fdgAbi = snapshots.LoadSuperFaultDisputeGameABI()
```

Comment:
I think op-dispute-mon will also need SuperAsteriscKonaGameType added in:
https://github.com/ethereum-optimism/optimism/blob/3085c7390302d0175b0f54b8a40bab5157920a49/op-dispute-mon/mon/extract/caller.go#L54-L67